pysrc/scripts/sampling.py

```diff
@@ -81,6 +94,37 @@ def clean_results(results_folder: str) -> None:
     for tmp_file in result_files:
         os.remove(os.path.join(os.getcwd(), results_folder, tmp_file))
 
+def get_File_Combinations(samples, config):
+    data_type_dict = {}
+    for num_files in range(0, len(samples)):
+        for sam_file in range(0, len(samples[num_files])):
+            if config.header:
+                particular_line = linecache.getline(samples[num_files][sam_file][0], 1)
+            else:
+                #TODO Get Next Lines if it was empty
+                particular_line = linecache.getline(samples[num_files][sam_file][0], 2)
+            particular_line = particular_line.split('\n')
+            if re.fullmatch(r'^[\d]+\.[\d]+', particular_line[0]):
+                dtype = "float"
+            elif  re.fullmatch(r'^[\d]+', particular_line[0]):
```

Comment:
```suggestion
            if re.fullmatch(r'^[\d]+\.[\d]+$', particular_line[0]):
                dtype = "float"
            elif  re.fullmatch(r'^[\d]+$', particular_line[0]):
```

To protect against lines like `1.0.0` (although I'm not sure they're possible).